content/direct-deposit-management/release-notes/2024-06-28.md

```diff
@@ -0,0 +1,5 @@
+We have added indicators for payment types in the response for the Direct Deposit API. This will support the ability to differentiate between different types of direct deposits.  
```

Comment:
```suggestion
We've added indicators for payment types in the response for the Direct Deposit API. This will support the ability to differentiate between different types of direct deposits.  
```